docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -398,7 +398,7 @@ These are provided in `application.properties` the same as any other config prop
 
 The properties are shown below:
 
-include::{generated-dir}/config/quarkus-core-native-config.adoc[opts=optional]
+include::{generated-dir}/config/quarkus-native-pkg-native-config.adoc[opts=optional]
```

Comment:
What's the rationale of this change?

Reply:
To use a generated `NativeConfig` root file. The file name is not handled derived from the package name by stripping out `runtime` or `deployment`.